### DISCUSSION TITLE
How can I use Rotating Proxies?

### DISCUSSION THREAD

**Discussion by Kenzogithubs** [KevinBytesTheDust/CVAmp#211] [Q&A]
Hello, I have rotating proxies with the form, how do I make it work?:

user-rotate:password@p.webshare.io:80

**Comment by KevinBytesTheDust** [comment]
1. Sign up at [Webshare.io](https://blueloperlabs.ch/proxy/wf) to receive **10 free proxies** plus **10% off** your first paid plan.  
2. Add the proxy entries directly into `proxy/proxy_list.txt` (see acquisition steps below).

Tip: Run CVAmp on a [Cloud Server with this guide](https://github.com/KevinBytesTheDust/CVAmp/wiki/Run-on-a-Cloud-Server), to stream lag-free.
